tests/test_ttr.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+
+import numpy as np
+
+from UncertainSCI.families import jacobi_weight_normalized, jacobi_recurrence_values
+from UncertainSCI.families import hermite_recurrence_values
+
+from UncertainSCI.mthd_mod_correct import compute_ttr_bounded, compute_ttr_unbounded
```

Comment:
Like Dan noticed, this generates an error -- there doesn't appear to be a `mthd_mod_correct` module in `UncertainSCI`, so this throws an error.

Reply:
Yes, I could not even remember where this came from. But anyway I delete them.